### DISCUSSION TITLE
Mapping existing SQLAlchemy ORMs to SQLModels

### DISCUSSION THREAD

**Discussion by havok2063** [fastapi/sqlmodel#658] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the SQLModel documentation, with the integrated search.
- [X] I already searched in Google "How to X in SQLModel" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to SQLModel but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to SQLModel but to [SQLAlchemy](https://github.com/sqlalchemy/sqlalchemy).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
N/A
```

### Description

I have an existing library of SQLAlchemy ORMs and I'm trying to build out a new FastAPI backend application.  Is there a way to map existing sqla ORMs into SQLModel models, to take advantage of the built-in syncing between Pydantic + SQLalchemy?

I started manually creating new Pydantic models for each ORM, but was wondering if I could use this package instead, without redefining my SQLAlchemy ORMs, which isn't an option for me.    Or is there a way to automap my sqla ORMs to Pydantic models?

### Operating System

macOS

### SQLModel Version

0.0.8

### Python Version

3.10

**Comment by MRigal** [comment]
I'm also curious to know (haven't tried out yet), if it would be possible to connect sqlalchemy's "Automap" module to create SQLModels out of a DB structure SQL file.

**Reply by joachimhuet** [reply]
If you build a tool like that I would be interested too 🍭 At the moment I'm doing a migration by rewriting all my models, it's so tiring
